examples/nextjs/pages/index.js

```diff
@@ -0,0 +1,207 @@
+import Head from 'next/head'
+
+export default function Home() {
+  return (
+    <div className='container'>
+      <Head>
+        <title>Create Next App</title>
+        <link rel="icon" href="/favicon.ico' /'
```

Comment:
Ahh, here is the problem. This line should be

```
<link rel="icon" href="/favicon.ico">
```

Reply:
Yes, it was the format step; this is from the pre-commit hook:

```
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{' '}
-          <img src="/vercel.svg" alt="Vercel Logo" className="logo" />
-        </a>
+          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app'
+          target='_blank'
+  rel = 'noopener noreferrer' >
+      Powered by{' '}<img src = '/vercel.svg' alt = 'Vercel Logo' className = 'logo' />
+      </a>
       </footer>

       <style jsx>{`
@@ -203,7 +201,7 @@ export default function Home() {
         * {
           box-sizing: border-box;
         }
-      `}</style>
-    </div>
+      `}</style><
+      /div>
   )
 }

      ERROR: please run yarn format to format changes in your commit
```